src/main/java/com/blamejared/crafttweaker/impl/brackets/BracketDumpers.java

```diff
@@ -124,4 +128,29 @@
                 .collect(Collectors.toList());
     }
     
+    @BracketDumper("damageSource")
+    public static Collection<String> getDamageSourceDump() {
+        return ExpandDamageSource.PRE_REGISTERED_DAMAGE_SOURCES.keySet()
+                .stream()
+                .map(name -> "<damageSource:" + name + ">")
+                .collect(Collectors.toList());
+    }
+    
+    @BracketDumper("equipmentSlotType")
```

Comment:
Ideally we want all BracketHandlers to be lowercase, so `damagesource` and `equipmentslottype`